ibis/sql/tests/test_compiler.py

```diff
@@ -2496,3 +2498,27 @@ def test_table_drop_with_filter():
     ON t0.`b` = t1.`b`
 WHERE t0.`a` < 1.0"""
     assert result == expected
+
+
+def test_large_compile():
```

Comment:
Sorry, I think it's better to move under `ibis/bigquery/tests/test_compiler.py` because this is testing the bigquery client